packages/frontend/core/src/modules/cloud/entities/subscription.ts

```diff
@@ -42,6 +42,11 @@ export class Subscription extends Entity {
       ? subscriptions.find(sub => sub.plan === SubscriptionPlan.AI)
       : null
   );
+  team$ = this.subscription$.map(subscriptions =>
```

Comment:
here this UserSubscription, you need a WorkspaceSubscription

Reply:
I can't seem to find the relevant backend interface, can you tell me which one it is?

Reply:
Open another PR to handle this issue

Reply:
packages/backend/server/src/schema.gql#978, it should be fixed in this PR, the logic is wrong